### PR TITLE
Adds autocomplete + search tests for SF Zoo being in SF

### PR DESCRIPTION
**Pull request by riordan**
Adds tests noted by @nvkelso in pelias/pelias#183

### CODE REVIEW

**Comment by orangejulius** [comment]
:+1:

File: test_cases/search.json [hunk 1]
```diff
@@ -599,6 +599,26 @@
           "region": "New Hampshire"
         }]
       }
+    },
+    {
+      "id": 24,
+      "status": "pass",
```
Comment (dianashk) [inline test_cases/search.json]: Please change fail if this isn't expected to pass at this time. Thanks.

Comment (riordan) [inline test_cases/search.json]: Yep; expect a barrage of PR's correcting this today.

**Comment by riordan** [comment]
@dianashk updated the tests here to failing. Should be ready to merge.

**Comment by dianashk** [comment]
:+1:

**Pull request merged**